innopoints/views/project.py

```diff
@@ -224,6 +329,12 @@ def patch(self, project_id):
             log.exception(err)
             abort(400, {'message': 'Data integrity violated.'})
 
+        if updated_project.review_status != old_status:
```

Comment:
How would it change? The`in_schema` only allows changing the name, image, organizer and mods.

Reply:
This was before you said it will have its dedicated endpoints